### PR TITLE
avoid namespace service and kind labels on node alerts

### PR DESCRIPTION
**Pull request by RoiGlinik**

### CODE REVIEW

File: src/robusta/core/reporting/base.py [hunk 1]
```diff
@@ -272,18 +272,21 @@ def __str__(self):
 
     def get_prometheus_silence_url(self, account_id: str, cluster_name: str) -> str:
         labels: Dict[str, str] = {"alertname": self.aggregation_key, "cluster": cluster_name, "account": account_id}
-        if self.subject.namespace:
-            labels["namespace"] = self.subject.namespace
+        kind: Optional[str] = self.subject.subject_type.value
+        if kind and kind == "node":
+            pass
+        else:
+            if self.subject.namespace:
```
Comment (Avi-Robusta) [inline src/robusta/core/reporting/base.py:279]: couldnt 
```
        if kind and kind == "node":
            pass
        else:
            if self.subject.namespace:
```
    be simplified as 
`if kind != "node" and self.subject.namespace:`

File: src/robusta/core/reporting/base.py [hunk 2]
```diff
@@ -272,18 +272,21 @@ def __str__(self):
 
     def get_prometheus_silence_url(self, account_id: str, cluster_name: str) -> str:
         labels: Dict[str, str] = {"alertname": self.aggregation_key, "cluster": cluster_name, "account": account_id}
-        if self.subject.namespace:
-            labels["namespace"] = self.subject.namespace
+        kind: Optional[str] = self.subject.subject_type.value
+        if kind and kind == "node":
+            pass
+        else:
```
Comment (Avi-Robusta) [inline src/robusta/core/reporting/base.py:278]: I would replace
```
        if kind and kind == "node":
            pass
        else:
```
with
` if kind != "node":`

File: src/robusta/core/reporting/base.py [hunk 3]
```diff
@@ -272,18 +272,21 @@ def __str__(self):
...
+            if self.subject.namespace:
+                labels["namespace"] = self.subject.namespace
 
-        if self.silence_labels and self.silence_labels.get("service"):
-            labels["service"] = self.silence_labels["service"]
+            if self.silence_labels and self.silence_labels.get("service"):
```
Comment (Avi-Robusta) [inline src/robusta/core/reporting/base.py:282]: can't you have service for a node?

File: src/robusta/core/reporting/base.py [hunk 4]
```diff
@@ -272,18 +272,21 @@ def __str__(self):
...
 
-        # In prometheus job is related to the scrape target.
-        # Kubernetes jobs are stored in job_name.
-        kind: Optional[str] = self.subject.subject_type.value
-        if kind and self.subject.name:
-            kind = "job_name" if kind == "job" else kind
-            labels[kind] = self.subject.name
+            # In prometheus, job is related to the scrape target.
+            # Kubernetes jobs are stored in job_name.
+            if kind and self.subject.name:
```
Comment (Avi-Robusta) [inline src/robusta/core/reporting/base.py:287]: are you sure you want to indent this, 
labels[kind] = self.subject.name
wont be run for nodes

Comment (RoiGlinik) [reply src/robusta/core/reporting/base.py:287]: Yes we want node alerts to be general, without a node name

File: src/robusta/core/reporting/base.py (hunk 2 above)
Comment (RoiGlinik) [reply src/robusta/core/reporting/base.py:278]: Yes I like it, ill change it

**Pull request merged**